src/plugins/kibana_react/public/page_template/page_template.tsx

```diff
@@ -142,6 +142,7 @@ export const KibanaPageTemplate: FunctionComponent<KibanaPageTemplateProps> = ({
           className: classNames(sideBarClasses, pageSideBarProps?.className),
         }}
         {...NO_DATA_PAGE_TEMPLATE_PROPS}
+        {...rest}
```

Comment:
@cchaos fyi on this change, I needed to pass these through to get the `data-test-subj` for tests.  Let me know if you see any issues with it

Reply:
Yeah unfortunately we can't do that because it will override other required settings of the no data page config. I actually have a fix for this in the Obs PR which you can emulate here. I can handle any conflict depending on which PR merges first. 

https://github.com/elastic/kibana/pull/107709/files#diff-17a88f2b6fa3ca9417c426095df517ea2aab6f31c6a1bc03049043b92fef5770

Reply:
got it - thanks for the github link!  I updated this PR with that change.

Reply:
👍  Thanks for the chage